controllers/nstemplateset/status.go

```diff
@@ -88,6 +91,37 @@ func (r *statusManager) setStatusReady(ctx context.Context, nsTmplSet *toolchain
 		})
 }
 
+func updateRevisions(nsTmplSet *toolchainv1alpha1.NSTemplateSet) {
+	nsTmplSet.Status.ClusterResources = nsTmplSet.Spec.ClusterResources
+	featureAnnotation, featureAnnotationFound := nsTmplSet.Annotations[toolchainv1alpha1.FeatureToggleNameAnnotationKey]
+	if featureAnnotationFound {
+		// save the feature toggles into the status
+		nsTmplSet.Status.FeatureToggles = utils.SplitCommaSeparatedList(featureAnnotation)
+	}
+	nsTmplSet.Status.Namespaces = nsTmplSet.Spec.Namespaces
+	nsTmplSet.Status.SpaceRoles = nsTmplSet.Spec.SpaceRoles
+}
+
+func (r *statusManager) updateStatusClusterResourcesRevisions(ctx context.Context, nsTmplSet *toolchainv1alpha1.NSTemplateSet) error {
+	nsTmplSet.Status.ClusterResources = nsTmplSet.Spec.ClusterResources
+	featureAnnotation, featureAnnotationFound := nsTmplSet.Annotations[toolchainv1alpha1.FeatureToggleNameAnnotationKey]
+	if featureAnnotationFound {
+		// save the feature toggles into the status
+		nsTmplSet.Status.FeatureToggles = utils.SplitCommaSeparatedList(featureAnnotation)
+	}
```

Comment:
it's currently used only for cluster-scoped resources, but could be technically used for namespace-scoped resources as well. 
it's worth adding a comment that it should be more generic and not tied to cluster-scoped template

Reply:
good point. done pls see: https://github.com/codeready-toolchain/member-operator/pull/648/commits/538ffec22b4abdaaf58db4964168e758504aae45